api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -287,6 +287,12 @@ message HttpConnectionManager {
   // is not desired it can be disabled.
   google.protobuf.BoolValue generate_request_id = 15;
 
+  // Whether the connection manager will keep the :ref:`x-request-id
+  // <config_http_conn_man_headers_x-request-id>` header if passed for a request that is edge
+  // (Edge request is the request from external clients to front Envoy) and not reset it, which
+  // is the current Envoy behaviour. This defaults to false.
+  google.protobuf.BoolValue edge_accept_request_id = 32;
```

Comment:
Drive by comment: `bool` will work here since the default is false.

Reply:
done